openstack/limes/templates/deployment-api.yaml

```
@@ -85,10 +85,10 @@ spec:
               # deployments. A full project report with `?detail` for that
               # domain is about 160 MiB of pure JSON.
               cpu: '1'
-              memory: '500Mi'
+              memory: '1Gi'
```

```suggestion
              memory: '700Mi'
```
lets first try a little less, to see if that helps enough already